SubmissionExample.ipynb

```diff
@@ -4,20 +4,20 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## End to End Mlperf Submission example\n",
+    "# End to End MLPerf Submission example\n",
     "\n",
-    "This is following the [General MLPerf Submission Rules](https://github.com/mlperf/policies/blob/master/submission_rules.adoc).\n"
+    "This is following the [General MLPerf Submission Rules](https://github.com/mlcommons/policies/blob/master/submission_rules.adoc).\n"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Get the mlperf source code\n",
+    "### Get the MLPerf Inference Benchmark Suite source code\n",
     "\n",
-    "You run this notebook from the root of the mlperf inference tree that you cloned with\n",
+    "You run this notebook from the root of the MLCommons Inference tree that you cloned with\n",
```

Comment:
The benchmarking part of mlcommons is still called mlperf.

Reply:
This reference was specific to which github repo to pull from (mlcommons/inference rather than mlperf/inference), but if you think it should be reverted, I'll defer to your expertise.

Reply:
got it.